tools/Utilities/Clauses.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Utilities
+{
+    public class Clauses
+    {
+        public string[] FrontMatter { get; set; } = Array.Empty<string>();
+        public string[] MainBody { get; set; } = Array.Empty <string>();
```

Comment:
Nit: remove space after `Empty`.
(I'm interested in your preference for arrays over `List<string>` here, but that can be something we discuss at leisure...)

Reply:
I suppose I could say something brilliant about these never being resized, only set during serialization, etc.

But, the truth is I used Visual Studio's "paste JSON text as class" and used that definition. 🔨